run_test

```diff
@@ -0,0 +1 @@
+false
```

Comment:
```suggestion
true
```
IMHO a sane default is probably `true`, since we want to run tests in majority of cases

Reply:
sure, that is also possible, but I am trying to get rid of this file and extra commit, probably we still need a commit to trigger a build, because I could not find in our configuration to build based on event for adding label